mubench.reviewsite/src/Controllers/RunsController.php

```diff
@@ -228,23 +233,36 @@ public static function exportRunStatistics($runs)
 
     function addRun($experimentId, $detectorId, $projectId, $versionId, $run)
     {
-        $detector = $this->createDetector($detectorId);
+        $detector = $this->getOrCreateDetector($detectorId);
         $experiment = Experiment::find($experimentId);
 
+        if($this->hasRunTable($detector)){
+            $saved_run = Run::of($detector)->in($experiment)->where(['project_muid' => $projectId, 'version_muid' => $versionId])->first();
+            if(intval($saved_run->timestamp) !== $run->{'timestamp'}){
```

Comment:
The table creation says this is an integer column, why do we need to cast it to integer?

Reply:
Apparently when using the normal mysql driver in php, it converts all columsn to strings when you select something from the database. This seems to also hold true with our in memory database for tests. Therefore the cast is needed.